File: testprojects/scala_main/pipelines/DONOT_OPEN_SCALA/code/src/main/scala/org/main/scla_dep_mgmt/graph/all_type_scala_sg_1/Limit_1_1.scala
package org.main.scla_dep_mgmt.graph.all_type_scala_sg_1

import io.prophecy.libs._
import org.main.scla_dep_mgmt.udfs.UDFs._
import org.main.scla_dep_mgmt.graph.all_type_scala_sg_1.config.Context
import org.apache.spark._
import org.apache.spark.sql._
import org.apache.spark.sql.functions._
import org.apache.spark.sql.types._
import org.apache.spark.sql.expressions._
import java.time._

object Limit_1_1 {

  def apply(context: Context, in: DataFrame): DataFrame =
    in.limit(context.config.c_sg1_c_limit)

}

File: testprojects/master_python_mainone/pipelines/StreamingPipelineMain/code/streamingpipelinemain/graph/all_components_main_1/Limit_1_1.py
from pyspark.sql import *
from pyspark.sql.functions import *
from pyspark.sql.types import *
from streamingpipelinemain.config.ConfigStore import *
from streamingpipelinemain.udfs.UDFs import *

def Limit_1_1(spark: SparkSession, in0: DataFrame) -> DataFrame:
    return in0.limit(100)
